src/app/dashboard/products/new/page.tsx

```
@@ -0,0 +1,225 @@
+    // Redirect
+    router.push('/dashboard/products');
+  };
+
+  return (
+    <div className="flex min-h-screen">
+      <main className="flex-1 p-6 ml-0 md:ml-64">
+        <div className="mb-4">
+          <button
+            onClick={() => setSidebarOpen(true)}
+            className="md:hidden text-orange-600 text-2xl"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)